docs/FEATURES.md

```diff
@@ -1,32 +1,75 @@
+| Fares                    | Zone-Based Fares          | One line of data in [areas.txt](https://gtfs.org/schedule/reference/#areastxt)   |     https://gtfs.org/getting_started/features/fares/#zone-based-fares |
+| Fares                    | Fares Transfers           | One line of data in [fare_transfer_rules.txt](https://gtfs.org/schedule/reference/#fare_transfer_rulestxt)    |    https://gtfs.org/getting_started/features/fares/#fares-transfers |
+| Fares                    | Fares V1                  | One line of data in [fare_attributes.txt](https://gtfs.org/schedule/reference/#fare_attributestxt)   |   https://gtfs.org/getting_started/features/fares/#fares-v1 |
+| Pathways                 | Pathways Connections*         | One line of data in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt)     |  https://gtfs.org/getting_started/features/pathways/#pathway-connections |
+| Pathways                 | Pathways Details*         | One value of **max_slope**<br/>OR<br/>**min_width** <br/>OR<br/>**length** <br/>OR<br/>**stair_count** in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt) | https://gtfs.org/getting_started/features/pathways/#pathway-details |
+| Pathways                 | Levels                    | One line of data in [levels.txt](https://gtfs.org/schedule/reference/#levelstxt)     |   https://gtfs.org/getting_started/features/pathways/#levels  |
+| Pathways                 | In-station traversal time | One **traversal_time** value in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt)   |   https://gtfs.org/getting_started/features/pathways/#in-station-traversal-time                                                                                                                                                                                                                                                                                                                                            |
+| Pathways                 | Pathway Signs       | One **signposted_as** value<br/>AND<br/>one **reversed_signposted_as** in [pathways.txt](https://gtfs.org/schedule/reference/#pathwaystxt) |       https://gtfs.org/getting_started/features/pathways/#pathway-signs     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
```

Comment:
This rule should be: "One `signposted_as` value **OR** one `reversed_signposted_as` in `pathways.txt`